respecConfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "specStatus": "ED",
+  "shortName": "vaccine-credentials-test-suite",
+  "subtitle": "Interoperability Report for Vaccine Credentials Data Model",
+  "github": "https://github.com/digitalbazaar/vaccination-certificate-test-suite",
+  "edDraftURI": "http://vaxcert-interop-reports.s3-website.us-east-2.amazonaws.com",
```

Comment:
Fix respec issue:
```suggestion
  "edDraftURI": "https://vaxcert-interop-reports.s3-website.us-east-2.amazonaws.com",
```

Reply:
Looks like that https correction doesn't work.  Maybe:
```
https://vaxcert-interop-reports.s3.us-east-2.amazonaws.com/index.html
```

Reply:
haha, why do you think S3 supports HTTPS?! :p 

https://docs.aws.amazon.com/AmazonS3/latest/userguide/WebsiteEndpoints.html

Reply:
The direct index.html https link above seems to work.  For this use case is the `s3-website` URL needed or could it use the https `s3` URL?